src/scss/themes/satellite.scss

```diff
@@ -590,6 +590,18 @@ $break-medium: 767px;
   font-style: normal;
 }
 
+/**
+ * ReverseHighlighting and ReverseSnippeting
+ */
+
+.ais-ReverseHighlight-highlighted,
+.ais-ReverseSnippet-highlighted {
+  color: $nebula500;
```

Comment:
Wondering if it's too flashy for inverted highlighting? Perhaps inheriting the color is fine to highlight differences.

Reply:
Good catch, it could be unreadable on light backgrounds.